perf-tests/servers/akka-http-only/src/main/scala/AkkaHttpOnlyServer.scala

```diff
@@ -0,0 +1,33 @@
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+
+import scala.io.StdIn
+
+import spray.json.DefaultJsonProtocol._
+
+final case class Book(id: Int)
+
+object AkkaHttpOnlyServer extends App {
+  implicit val actorSystem = ActorSystem(Behaviors.empty, "akka-http")
+  implicit val executionContext = actorSystem.executionContext
+  implicit val bookMarshaller: spray.json.RootJsonFormat[Book] = jsonFormat1(Book.apply)
```

Comment:
we are using different json libraries so there's the danger that we are also testing performance differences between those two. Maybe it would make more sense to test two endpoints which return plain text answers, sth like a `Hello world`

Reply:
done